pkg/plugin/probes-handler/probes_handler.go

```diff
@@ -0,0 +1,18 @@
+package probeshandler
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+// NewProbesHandler registers liveliness and readinesss probes for /version endpoint
+func NewProbesHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		_, err := w.Write([]byte(os.Getenv("VERSION")))
```

Comment:
@hemanik How about returning `version: 406ea76-1524559762` instead of only `406ea76-1524559762`?

Reply:
Can we respond using proper json, content type and so on?